wordpress-mobile/WordPress-Android
# Issue/11816 feature announcement analytics

Fixes #11816

This PR adds analytics to Feature Announcement.

We are tracking:
- When the feature announcement is shown on app upgrade.
- When the feature announcement is shown from app settings.
- When find out more button is tapped.
- When the continue button is tapped an how much time user spent on the screen before tapping it.

To test:
- Ideally, place breakpoints near the events and make sure they are called when expected.
- Pay special attention to the time tracking part, and confirm that it matches the time spent on screen.

PR submission checklist:
